helix-term/src/commands.rs

```diff
@@ -587,6 +587,7 @@ fn move_impl(cx: &mut Context, move_fn: MoveFn, dir: Direction, behaviour: Movem
             &mut annotations,
         )
     });
+    drop(annotations);
```

Comment:
A comment why the drop is necessary would be nice :)

Reply:
It's just a borrow check/lifetime inference limitation, so basically just making the compiler shutup. If inference was better this wouldn't be needed. In this case I would need to dig into what exact rustc feature would be necessary for the compiler to automatically infer the right drop order and the explanation would be quite long so I don't think that makes sense. I can add a `// make borrow check happy` comment but there is no functional reason for this `drop` so I don't think it really warrants anything more

Reply:
I thought it was that but I was wondering, you can resolve then :) (no being able to resolve my own comments is strange 🤔)